bundle/edu.gemini.sp.vcs/src/main/scala/edu/gemini/sp/vcs2/VcsServer.scala

```diff
@@ -100,6 +100,23 @@ class VcsServer(odb: IDBDatabaseService) { vs =>
     }
   }
 
+  /** Replaces the program in the database whose key and id match program the
+    * given program with program p.
```

Comment:
Minor issue, but we can ignore it, I suppose:

"match program the given program"